<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotObviousMethod</class>
 <widget class="QWidget" name="NotObviousMethod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>961</width>
     <height>621</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calculateButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="equationA">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="equationB">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="accurateA">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="accurateB">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
